Formula/stone-soup.rb

```diff
@@ -26,6 +26,10 @@ class StoneSoup < Formula
     depends_on "freetype"
   end
 
+  if build.with? "lua"
```

Comment:
You can kill the `if build.with? "lua"` and `end`. Just the `depends_on "lua@5.1" => :recommended` needs to remain with the tweaked option text.